source/drivers/driver-compatibility-reference.txt

```diff
@@ -513,37 +523,63 @@ Language Compatibility
 .. list-table::
    :header-rows: 1
    :stub-columns: 1
-   :class: compatibility
+   :class: compatibility-large
 
    * - PHP Driver
      - PHP 5.3
      - PHP 5.4
      - PHP 5.5
      - PHP 5.6
+     - PHP 7.0
+     - HHVM
```

Comment:
We're going to have to start to be more specific here too. Right now, it should say "HHVM 3.9".